test/infrastructure/container/docker.go

```diff
@@ -320,10 +582,22 @@ func getProxyDetails() *proxyDetails {
 		if val == "" {
 			continue
 		}
+		proxySupport = true
 		details.Envs[name] = val
 		details.Envs[strings.ToLower(name)] = val
 	}
 
+	// Specifically add the docker network subnets to NO_PROXY if we are using proxies
+	if proxySupport {
+		subnets, err := d.getSubnets(ctx, network)
+		if err != nil {
+			return &details
```

Comment:
Is it really okay to silently handle the error this way? If yes, can you add a comment why, please.